Sources/Hedera/ChunkedTransaction.swift

```diff
@@ -260,16 +264,16 @@ extension ChunkedTransaction.ChunkView: Execute {
     ) {
         assert(transaction.isFrozen)
 
-        guard let transactionId = transactionId else {
-            throw HError.noPayerAccountOrTransactionId
-        }
+        let currentTransactionId = TransactionId(
+            accountId: initialTransactionId.accountId,
+            validStart: initialTransactionId.validStart.adding(nanos: UInt64(currentChunk)))
```

Comment:
`generateFromInitial` iff the explicit transaction ID for the chunk doesn't exist

If we get a transaction from `fromBytes` that has custom transaction IDs on every chunk because the user in some other SDK is capable of setting that (or a hypothetical future version of this SDK), we should respect it